src/tools/rbd/action/Export.cc

```diff
@@ -17,10 +17,6 @@
 
 namespace rbd {
 namespace action {
-namespace export_diff {
```

Comment:
Instead of leaking these methods into the ```action``` namespace, I would combine the ```export_diff``` and ```export_full``` namespaces (and rename methods where needed). This comment applies to the next commit for the ```import_diff``` namespace removal as well.